test/json/snapshots/test.js.md

```diff
@@ -81,9 +117,6 @@ Generated by [AVA](https://ava.li).
       'a.world',
       'b.hello',
       'b.world',
-      'c.hello',
-      'y.hello',
-      'z.hello',
```

Comment:
Why diff?

Reply:
damn, forgot to fix that, sorry. I'll look into it.